### ISSUE TITLE
R-squared in SUR.from_formula

### ISSUE THREAD

**Issue by tra6sdc** [bashtage/linearmodels#215]
Is the r2 reported by SUR unadjusted? I located these lines of code:

````
stats['resid_ss'] = float(resid[:, [i]].T @ resid[:, [i]])
stats['total_ss'] = total_ss
stats['r2'] = 1.0 - stats.resid_ss / stats.total_ss
stats['r2a'] = 1.0 - (stats.resid_ss / df_r) / (stats.total_ss / df_c)
````
but r2a appears not to be available. 

Also some have used McElroy's R2 for SUR. [https://stats.stackexchange.com/questions/93751/goodness-of-fit-of-a-sur-model](https://stats.stackexchange.com/questions/93751/goodness-of-fit-of-a-sur-model). I think that all the information is available to calculate this quantity? Thanks.

**Comment by bashtage** [comment]
Probably an oversight. Will add as an enhancement request.

**Comment by bashtage** [comment]
Closed in #225
